Index.html

```diff
@@ -186,5 +186,253 @@ <h2 class="hero-text-title">I'm a Full Stack Web Developer</h2>
       </div>
 
     </section>
+
+    <section class="recent-works" id="recent-works-id">
+
+      <h2 class="recent-works-title"><span>My Recent Works</span></h2>
+
+      <div class="main-work-box">
+        <div class="main-work-image"></div>
+        <div class="main-work-text-box">
+          <h3 class="main-work-text-title">Lifestyle-Articles Web Application</h3>
+          <p class="main-work-text-area">
+          Based on the website design of <a href="https://www.behance.net/gallery/14554909/liFEsTlye-Mobile-version" class="nelson-sakwa">Nelson Sakwa on Behance</a>. Build with Ruby on Rails, the project has the models, views and controllers of each of the resources User, Article, Vote and Category. The User has authentication and can perform any of the CRUD operations on the Articles and can create and delete a Vote. Article and Category has a many to many relationship. The Article with the most Vote will be displayed in the hero section of the main page.
+          </p>
+          <div class="main-work-text-skills-box">
+            <span class="skill">HTML</span>
+            <span class="skill">CSS</span>
+            <span class="skill">Ruby</span>
+            <span class="skill">Rails</span>
+          </div>
+          <a class="see-project-button" href="https://lifestyle-articles-jose-abel.herokuapp.com/">See Project</a>
+        </div>
+
+      </div>
+
+      <div class="recent-works-row">
+
+        <div class="recent-works-col recent-works-col-1">
+          <h3 class="recent-works-col-title">Social Media App</h3>
+          <p class="recent-works-col-text">
+            Social media web app that has the entities of Users, Posts, Comments, Likes, Friendships associations and authentication implementations.
+          </p>
+          <div class="recent-works-skills-box">
+            <span class="recent-works-col-skills">HTML</span>
+            <span class="recent-works-col-skills">CSS</span>
+            <span class="recent-works-col-skills">RAILS</span>
+            <a class="card-button" href="https://guillain-jose-social-media-app.herokuapp.com">See Project</a>
+          </div>
+        </div>
```

Comment:
- [ ] Please make sure the project cards match the Figma design.
   - Please make sure the card content is aligned to the left instead of the center.
   - Please remove the border from the project technologies and apply the correct background color.
   - Please make sure the `See project` button is available without hovering too and looks like the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166122824-abdcc758-8e37-4d67-9d3f-d28fb3fa8fb1.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166122829-c892de78-69c8-468c-9d6b-90f2d2a7543c.png)